<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>788</width>
    <height>618</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>400</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Unreal</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>400</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>matlab</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_3">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>400</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>visual Studio</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="textBrowser">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>60</y>
      <width>321</width>
      <height>101</height>
     </rect>
    </property>
    <property name="html">
     <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'SimSun'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p align=&quot;center&quot; style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:12pt; font-weight:600;&quot;&gt;Welcome to AU simulation platform!&lt;/span&gt;&lt;/p&gt;
&lt;p align=&quot;center&quot; style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:12pt; font-weight:600;&quot;&gt;Please set the right path to start!&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>250</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string extracomment="Unreal Path">Unreal</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Airsim</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Matlab</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Visual Studio</string>
     </property>
    </item>
   </widget>
   <widget class="QLineEdit" name="lineEdit">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>250</y>
      <width>341</width>
      <height>31</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>please input the right path here</string>
    </property>
   </widget>
   <widget class="QToolButton" name="toolButton">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>250</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>...</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_4">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>510</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>exit</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>788</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>files</string>
    </property>
    <addaction name="separator"/>
    <addaction name="actionpath_setting"/>
    <addaction name="actionopen_local_files"/>
    <addaction name="actioncopy"/>
    <addaction name="actionpaste"/>
    <addaction name="actionsave"/>
    <addaction name="actionsave_as"/>
   </widget>
   <widget class="QMenu" name="menuwork">
    <property name="title">
     <string>work</string>
    </property>
    <addaction name="actionworkplace"/>
   </widget>
   <widget class="QMenu" name="menusetting">
    <property name="title">
     <string>setting</string>
    </property>
    <addaction name="actionDrone_setting"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menuwork"/>
   <addaction name="menusetting"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionworkplace">
   <property name="text">
    <string>workplace</string>
   </property>
  </action>
  <action name="actionpath_setting">
   <property name="text">
    <string>new file</string>
   </property>
  </action>
  <action name="actionopen_local_files">
   <property name="text">
    <string>open file</string>
   </property>
  </action>
  <action name="actionDrone_setting">
   <property name="text">
    <string>path setting</string>
   </property>
  </action>
  <action name="actioncopy">
   <property name="text">
    <string>copy</string>
   </property>
  </action>
  <action name="actionpaste">
   <property name="text">
    <string>paste</string>
   </property>
  </action>
  <action name="actionsave">
   <property name="text">
    <string>save</string>
   </property>
  </action>
  <action name="actionsave_as">
   <property name="text">
    <string>save as</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
